### PR TITLE
Manage tls defaults

### PR DESCRIPTION
**Pull request by alxwr**
**Summary of Changes**
 * corrected some `map.jinja` values
 * use `{{ configdir }}/tls-defaults.conf` to override the distributions default values
 * use up-to-date cryptographic parameters according to bettercrypto.org
 * added Pillar `apache:mod_ssl` to opt-in and alter default values
 * documented in `pillar.example`

**Tested with**
 * FreeBSD 10.3 and Ubuntu 16.04.
 *  `testssl.sh` (https://testssl.sh/) and https://www.ssllabs.com/ssltest/analyze.html

### CODE REVIEW

**Comment by alxwr** [comment]
@arthurlogilab Please test this PR against your setup!

File: apache/files/FreeBSD/apache-2.4.config.jinja [hunk 1]
```diff
@@ -526,5 +526,9 @@ TraceEnable Off
 # Well, IncludeOptional behaved lile Include
 IncludeOptional etc/apache24/extra/security.con[f]
 
+{% if salt['pillar.get']('apache:mod_ssl:manage_tls_defaults', False) -%}
+Include etc/apache24/extra/tls-defaults.conf
```
Comment (javierbertoli) [inline apache/files/FreeBSD/apache-2.4.config.jinja]: I don't have any FreeBSD around (so I don't know if these paths should be relative or absolute), but isn't this line missing an initial '/' in the `Include` path?

Comment (aboe76) [reply apache/files/FreeBSD/apache-2.4.config.jinja]: looks like the configuration from:
http://www.xfiles.dk/guide-on-how-to-configure-the-apache-24-httpd-conf-file-on-freebsd/

Comment (javierbertoli) [reply apache/files/FreeBSD/apache-2.4.config.jinja]: Yeah, reading the ` apache/map.jinja`, it seems all these files are relative to the `/usr/local` dir.

**Comment by javierbertoli** [comment]
Thanks @alxwr and @aboe76!

**Pull request merged**